packages/docs-reanimated/docs/guides/worklets.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: worklets
+sidebar_label: Worklets
+sidebar_position: 1
+---
+
+# Worklets
+
+Worklets are short-running JavaScript functions that can be run on the [UI thread](/docs/fundamentals/glossary#ui-thread). Reanimated uses worklets to update view properties and react to events on the UI thread.
```

Comment:
1. Not only on the UI thread (also on different threads), but mainly in other JS runtimes than the React runtime.
2. Reanimated doesn't use worklets to update view properties, this is handled mostly by native code. Reanimated uses worklets to calculate view styles for the current frame. Also, Reanimated can run worklets as a reaction to some event while still on the UI thread.